Build error: ectrans@1.5.0 with ifx 2025.0.0

**Describe the bug**
(please read all the way to the end of the description)
```
[+] /home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/py-setuptools-63.4.3-qoc6roh
==> Installing ectrans-1.5.0-wdbdj3oltmlcx3on2wljmzvbo73dx5e6 [148/190]
==> No binary for ectrans-1.5.0-wdbdj3oltmlcx3on2wljmzvbo73dx5e6 found: installing from source
==> Fetching https://mirror.spack.io/_source-cache/archive/8b/8b2b24d1988b92dc3793b29142946614fca9e9c70163ee207d2a123494430fde.tar.gz
==> No patches needed for ectrans
==> ectrans: Executing phase: 'cmake'
==> [2024-12-13-19:55:46.000690] '/usr/bin/cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ectrans-1.5.0-wdbdj3o' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=ON' '-DCMAKE_INSTALL_RPATH:STRING=/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ectrans-1.5.0-wdbdj3o/lib;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ectrans-1.5.0-wdbdj3o/lib64' '-DCMAKE_PREFIX_PATH:STRING=/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/fiat-1.4.1-yewylrl;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/fckit-0.13.2-t5agfxx;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/python-3.11.7-gxvuvnm;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/xz-5.4.6-gfxehlr;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/util-linux-uuid-2.40.2-f2ugvfa;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/sqlite-3.46.0-sx772zd;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/openssl-3.4.0-yhkonvb;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/zlib-ng-2.2.1-5ylidht;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/libxcrypt-4.4.35-urkfj6y;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/libffi-3.4.6-sk5zqh6;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/gdbm-1.24-ptczk2s;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/readline-8.2-sjantly;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/expat-2.6.4-c3mjcsh;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/eckit-1.28.3-xgbvdxz;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/ncurses-6.5-luybn7y;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/lz4-1.10.0-3tr43ax;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/eigen-3.4.0-hh53g3j;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/bzip2-1.0.8-67mprh2;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ecbuild-3.7.2-uhlispg;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/gcc/11.5.0/gcc-runtime-11.5.0-tlhi2ue;/opt/intel/oneapi;/opt/intel/oneapi;/opt/intel/oneapi' '-DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_POLICY_DEFAULT_CMP0090:STRING=NEW' '-DCMAKE_FIND_USE_PACKAGE_REGISTRY:BOOL=OFF' '-DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON' '-DENABLE_MPI:BOOL=ON' '-DENABLE_OMP:BOOL=ON' '-DENABLE_DOUBLE_PRECISION:BOOL=ON' '-DENABLE_SINGLE_PRECISION:BOOL=ON' '-DENABLE_FFTW:BOOL=OFF' '-DENABLE_MKL:BOOL=ON' '-DENABLE_TRANSI:BOOL=ON' '/home/dom/work/spack-stack/spst-neptune-ops/cache/build_stage/spack-stage-ectrans-1.5.0-wdbdj3oltmlcx3on2wljmzvbo73dx5e6/spack-src'
-- The C compiler identification is IntelLLVM 2025.0.0
-- The CXX compiler identification is IntelLLVM 2025.0.0
-- The Fortran compiler identification is IntelLLVM 2025.0.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/dom/work/spack-stack/spst-neptune-ops/spack/lib/spack/env/oneapi/icx - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/dom/work/spack-stack/spst-neptune-ops/spack/lib/spack/env/oneapi/icpx - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting Fortran compiler ABI info
-- Detecting Fortran compiler ABI info - done
-- Check for working Fortran compiler: /home/dom/work/spack-stack/spst-neptune-ops/spack/lib/spack/env/oneapi/ifx - skipped
-- ecbuild   3.7.2      /home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ecbuild-3.7.2-uhlispg/share/ecbuild/cmake
-- cmake     3.26.5     /usr/bin/cmake
-- ---------------------------------------------------------
-- Found Git: /usr/bin/git (found version "2.43.5")
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of void*
-- Check size of void* - done
-- Check size of off_t
-- Check size of off_t - done
-- ---------------------------------------------------------
-- ---------------------------------------------------------
-- [ectrans] (1.5.0)
-- Feature TESTS enabled
-- Found OpenMP_Fortran: -fiopenmp
-- Found OpenMP: TRUE  found components: Fortran
-- ectrans FOUND fiat: /home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/fiat-1.4.1-yewylrl/lib64/cmake/fiat (found version "1.4.1")
-- Found MPI_CXX: /opt/intel/oneapi/mpi/2021.14/lib/libmpicxx.so (found version "3.1")
-- Found MPI_Fortran: /opt/intel/oneapi/mpi/2021.14/lib/libmpifort.so (found version "3.1")
-- Found MPI: TRUE (found version "3.1") found components: Fortran CXX
-- ectrans FOUND MPI
--    MPI_LIBRARIES : [/opt/intel/oneapi/mpi/2021.14/lib/libmpicxx.so /opt/intel/oneapi/mpi/2021.14/lib/libmpifort.so /opt/intel/oneapi/mpi/2021.14/lib/libmpi.so /usr/lib64/libdl.a /usr/lib64/librt.a /usr/lib64/libpthread.a]
-- Found package MPI required for feature MPI
-- Feature MPI enabled
-- Found OpenMP_Fortran: -fiopenmp
-- ectrans FOUND OpenMP
-- Found package OpenMP required for feature OMP
-- Feature OMP enabled
-- Feature DOUBLE_PRECISION enabled
-- Feature SINGLE_PRECISION enabled
-- Found package MKL required for feature MKL
-- Feature MKL enabled
-- Feature CPU enabled
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.7.3")
-- FFTW targets:
--     FFTW::fftw3 (double precision)  [/opt/intel/oneapi/mkl/2025.0/lib/libmkl_intel_lp64.so;/opt/intel/oneapi/mkl/2025.0/lib/libmkl_sequential.so;/opt/intel/oneapi/mkl/2025.0/lib/libmkl_core.so]
--     FFTW::fftw3f (single precision)  [/opt/intel/oneapi/mkl/2025.0/lib/libmkl_intel_lp64.so;/opt/intel/oneapi/mkl/2025.0/lib/libmkl_sequential.so;/opt/intel/oneapi/mkl/2025.0/lib/libmkl_core.so]
-- Found FFTW: /opt/intel/oneapi/mkl/2025.0/include/fftw  found components: double single
-- ectrans FOUND FFTW
--    FFTW_INCLUDE_DIRS : [/opt/intel/oneapi/mkl/2025.0/include/fftw]
--    FFTW_LIBRARIES : [/opt/intel/oneapi/mkl/2025.0/lib/libmkl_intel_lp64.so /opt/intel/oneapi/mkl/2025.0/lib/libmkl_sequential.so /opt/intel/oneapi/mkl/2025.0/lib/libmkl_core.so]
-- Feature TRANSI enabled
-- Feature CUTLASS_3XTF32 was not enabled (also not requested) -- following condition was not met: HAVE_SINGLE_PRECISION AND HAVE_CUTLASS
-- Feature GPU_AWARE_MPI was not enabled (also not requested) -- following condition was not met: HAVE_GPU
-- Feature GPU_GRAPHS_GEMM was not enabled (also not requested) -- following condition was not met: HAVE_GPU
-- Feature GPU_GRAPHS_FFT was not enabled (also not requested) -- following condition was not met: HAVE_GPU
-- Found MPI: TRUE (found version "3.1") found components: Fortran
-- ectrans FOUND MPI
--    MPI_LIBRARIES : [/opt/intel/oneapi/mpi/2021.14/lib/libmpicxx.so /opt/intel/oneapi/mpi/2021.14/lib/libmpifort.so /opt/intel/oneapi/mpi/2021.14/lib/libmpi.so /usr/lib64/libdl.a /usr/lib64/librt.a /usr/lib64/libpthread.a]
-- Found package MPI required for feature TESTS
-- Feature TESTS enabled
-- Performing Test ectrans_Fortran_FLAG_TEST_1
-- Performing Test ectrans_Fortran_FLAG_TEST_1 - Success
-- Added Fortran flag [-march=core-avx2 -no-fma] to build type BIT
-- Performing Test ectrans_Fortran_FLAG_TEST_2
-- Performing Test ectrans_Fortran_FLAG_TEST_2 - Failed
CMake Error at /home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ecbuild-3.7.2-uhlispg/share/ecbuild/cmake/ecbuild_log.cmake:190 (message):
  CRITICAL - Fortran compiler
  /home/dom/work/spack-stack/spst-neptune-ops/spack/lib/spack/env/oneapi/ifx
  does not recognise Fortran flag '-fast-transcendentals -fp-model precise
  -fp-speculation=safe'
Call Stack (most recent call first):
  /home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ecbuild-3.7.2-uhlispg/share/ecbuild/cmake/ecbuild_add_lang_flags.cmake:116 (ecbuild_critical)
  /home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ecbuild-3.7.2-uhlispg/share/ecbuild/cmake/ecbuild_add_fortran_flags.cmake:44 (ecbuild_add_lang_flags)
  cmake/ectrans_compile_options.cmake:30 (ecbuild_add_fortran_flags)
  CMakeLists.txt:168 (include)


-- Configuring incomplete, errors occurred!
==> Error: ProcessError: Command exited with status 1:
    '/usr/bin/cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ectrans-1.5.0-wdbdj3o' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=ON' '-DCMAKE_INSTALL_RPATH:STRING=/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ectrans-1.5.0-wdbdj3o/lib;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ectrans-1.5.0-wdbdj3o/lib64' '-DCMAKE_PREFIX_PATH:STRING=/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/fiat-1.4.1-yewylrl;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/fckit-0.13.2-t5agfxx;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/python-3.11.7-gxvuvnm;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/xz-5.4.6-gfxehlr;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/util-linux-uuid-2.40.2-f2ugvfa;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/sqlite-3.46.0-sx772zd;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/openssl-3.4.0-yhkonvb;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/zlib-ng-2.2.1-5ylidht;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/libxcrypt-4.4.35-urkfj6y;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/libffi-3.4.6-sk5zqh6;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/gdbm-1.24-ptczk2s;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/readline-8.2-sjantly;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/expat-2.6.4-c3mjcsh;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/eckit-1.28.3-xgbvdxz;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/ncurses-6.5-luybn7y;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/lz4-1.10.0-3tr43ax;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/eigen-3.4.0-hh53g3j;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-ops-oneapi-2025.0.0/install/oneapi/2025.0.0/bzip2-1.0.8-67mprh2;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ecbuild-3.7.2-uhlispg;/home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/gcc/11.5.0/gcc-runtime-11.5.0-tlhi2ue;/opt/intel/oneapi;/opt/intel/oneapi;/opt/intel/oneapi' '-DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_POLICY_DEFAULT_CMP0090:STRING=NEW' '-DCMAKE_FIND_USE_PACKAGE_REGISTRY:BOOL=OFF' '-DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON' '-DENABLE_MPI:BOOL=ON' '-DENABLE_OMP:BOOL=ON' '-DENABLE_DOUBLE_PRECISION:BOOL=ON' '-DENABLE_SINGLE_PRECISION:BOOL=ON' '-DENABLE_FFTW:BOOL=OFF' '-DENABLE_MKL:BOOL=ON' '-DENABLE_TRANSI:BOOL=ON' '/home/dom/work/spack-stack/spst-neptune-ops/cache/build_stage/spack-stage-ectrans-1.5.0-wdbdj3oltmlcx3on2wljmzvbo73dx5e6/spack-src'

1 error found in build log:
     73    -- Feature TESTS enabled
     74    -- Performing Test ectrans_Fortran_FLAG_TEST_1
     75    -- Performing Test ectrans_Fortran_FLAG_TEST_1 - Success
     76    -- Added Fortran flag [-march=core-avx2 -no-fma] to build type BIT
     77    -- Performing Test ectrans_Fortran_FLAG_TEST_2
     78    -- Performing Test ectrans_Fortran_FLAG_TEST_2 - Failed
  >> 79    CMake Error at /home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ecbuild-3.7.2-uhlispg/share/ecbuild/cmake/ecbuild_log.cmake:190 (message):
     80      CRITICAL - Fortran compiler
     81      /home/dom/work/spack-stack/spst-neptune-ops/spack/lib/spack/env/oneapi/ifx
     82      does not recognise Fortran flag '-fast-transcendentals -fp-model precise
     83      -fp-speculation=safe'
     84    Call Stack (most recent call first):
     85      /home/dom/work/spack-stack/spst-neptune-ops/envs/ne-dev-oneapi-2025.0.0/install/oneapi/2025.0.0/ecbuild-3.7.2-uhlispg/share/ecbuild/cmake/ecbuild_add_lang_flags.cmake:116 (ecbuild_critical)

See build log for details:
  /home/dom/work/spack-stack/spst-neptune-ops/cache/build_stage/spack-stage-ectrans-1.5.0-wdbdj3oltmlcx3on2wljmzvbo73dx5e6/spack-build-out.txt
```

**To Reproduce**
Install oneAPI 2025.0.0, configure `oneapi` compiler to use `ifx`, then build an environment that contains `ectrans`

**Expected behavior**
No build errors

**System:**
@climbfuji's dev system with OneAPI 2025.0.0

**Additional context**
`ectrans` was recently update from 1.2.0 to 1.5.0. The error occurs with 1.5.0 and 1.4.0 (1.3.0 is not available in spack). ectrans@1.2.0 builds fine. We need to fix this error or roll back to 1.2.0 for spack-stack-1.9.0. Fortunately, the newest versions of eckit, fckit, ecmwf-atlas all build fine with ectrans@1.2.0 (I don't know if there are any runtime version incompatibilities, though).

--- a/configs/common/packages.yaml
+++ b/configs/common/packages.yaml
@@ -55,7 +55,7 @@ packages:
     require: '@0.40.0 +fckit +trans +tesselation +fftw'
   ectrans:
     require:
-    - '@1.5.0'
+    - '@1.2.0'
   eigen:
     require: '@3.4.0'
   # Attention - when updating the version also check the common modules.yaml

--- a/configs/containers/specs/jedi-ci.yaml
+++ b/configs/containers/specs/jedi-ci.yaml
@@ -15,7 +15,7 @@
     eckit@1.28.3,
     ecmwf-atlas@0.40.0 +fckit +trans +tesselation +fftw,
     fiat@1.4.1,
-    ectrans@1.5.0 +fftw,
+    ectrans@1.2.0 +fftw,
     eigen@3.4.0,
     fckit@0.13.2,
     fms@2024.02,

--- a/configs/sites/tier1/acorn/packages_intel.yaml
+++ b/configs/sites/tier1/acorn/packages_intel.yaml
@@ -25,7 +25,7 @@
     # DH* Remove this section to switch to intel-oneapi-mkl
     ectrans:
       require::
-      - '@1.5.0 ~mkl +fftw'
+      - '@1.2.0 ~mkl +fftw'
     gsibec:
       require::
       - '@1.2.1 ~mkl'

--- a/configs/sites/tier1/aws-pcluster/packages_intel.yaml
+++ b/configs/sites/tier1/aws-pcluster/packages_intel.yaml
@@ -25,7 +25,7 @@ packages:
   # DH* Remove this section to switch to intel-oneapi-mkl
   ectrans:
     require::
-    - '@1.5.0 ~mkl +fftw'
+    - '@1.2.0 ~mkl +fftw'
   gsibec:
     require::
     - '@1.2.1 ~mkl'

--- a/configs/sites/tier1/gaea-c5/packages.yaml
+++ b/configs/sites/tier1/gaea-c5/packages.yaml
@@ -29,7 +29,7 @@ packages:
   # DH* Remove this section to switch to intel-oneapi-mkl
   ectrans:
     require::
-    - '@1.5.0 ~mkl +fftw'
+    - '@1.2.0 ~mkl +fftw'
   gsibec:
     require::
     - '@1.2.1 ~mkl'

--- a/configs/sites/tier1/gaea-c6/packages.yaml
+++ b/configs/sites/tier1/gaea-c6/packages.yaml
@@ -28,7 +28,7 @@ packages:
   # DH* Remove this section to switch to intel-oneapi-mkl
   ectrans:
     require::
-    - '@1.5.0 ~mkl +fftw'
+    - '@1.2.0 ~mkl +fftw'
   gsibec:
     require::
     - '@1.2.1 ~mkl'

--- a/configs/sites/tier1/hera/packages_intel.yaml
+++ b/configs/sites/tier1/hera/packages_intel.yaml
@@ -24,7 +24,7 @@ packages:
   # DH* Remove this section to switch to intel-oneapi-mkl
   ectrans:
     require::
-    - '@1.5.0 ~mkl +fftw'
+    - '@1.2.0 ~mkl +fftw'
   gsibec:
     require::
     - '@1.2.1 ~mkl'

--- a/configs/sites/tier1/hercules/packages_intel.yaml
+++ b/configs/sites/tier1/hercules/packages_intel.yaml
@@ -24,7 +24,7 @@ packages:
   # Remove this section to switch to intel-oneapi-mkl
   ectrans:
     require::
-    - '@1.5.0 ~mkl +fftw'
+    - '@1.2.0 ~mkl +fftw'
   gsibec:
     require::
     - '@1.2.1 ~mkl'

--- a/configs/sites/tier1/nautilus/packages_aocc.yaml
+++ b/configs/sites/tier1/nautilus/packages_aocc.yaml
@@ -26,7 +26,7 @@ packages:
     buildable: False
   ectrans:
     require::
-    - '@1.5.0 +mkl ~fftw'
+    - '@1.2.0 +mkl ~fftw'
   gsibec:
     require::
     - '@1.2.1 +mkl'

--- a/configs/sites/tier1/nautilus/packages_gcc.yaml
+++ b/configs/sites/tier1/nautilus/packages_gcc.yaml
@@ -25,7 +25,7 @@ packages:
     buildable: False
   ectrans:
     require::
-    - '@1.5.0 +mkl ~fftw'
+    - '@1.2.0 +mkl ~fftw'
   gsibec:
     require::
     - '@1.2.1 +mkl'

--- a/configs/sites/tier1/noaa-aws/packages_intel.yaml
+++ b/configs/sites/tier1/noaa-aws/packages_intel.yaml
@@ -24,7 +24,7 @@ packages:
   # DH* Remove this section to switch to intel-oneapi-mkl
   ectrans:
     require::
-    - '@1.5.0 ~mkl +fftw'
+    - '@1.2.0 ~mkl +fftw'
   gsibec:
     require::
     - '@1.2.1 ~mkl'

--- a/configs/sites/tier1/noaa-azure/packages_intel.yaml
+++ b/configs/sites/tier1/noaa-azure/packages_intel.yaml
@@ -24,7 +24,7 @@ packages:
   # DH* Remove this section to switch to intel-oneapi-mkl
   ectrans:
     require::
-    - '@1.5.0 ~mkl +fftw'
+    - '@1.2.0 ~mkl +fftw'
   gsibec:
     require::
     - '@1.2.1 ~mkl'

--- a/configs/sites/tier1/noaa-gcloud/packages_intel.yaml
+++ b/configs/sites/tier1/noaa-gcloud/packages_intel.yaml
@@ -24,7 +24,7 @@ packages:
   # DH* Remove this section to switch to intel-oneapi-mkl
   ectrans:
     require::
-    - '@1.5.0 ~mkl +fftw'
+    - '@1.2.0 ~mkl +fftw'
   gsibec:
     require::
     - '@1.2.1 ~mkl'

--- a/configs/sites/tier1/orion/packages_intel.yaml
+++ b/configs/sites/tier1/orion/packages_intel.yaml
@@ -24,7 +24,7 @@ packages:
   # Remove this section to switch to intel-oneapi-mkl
   ectrans:
     require::
-    - '@1.5.0 ~mkl +fftw'
+    - '@1.2.0 ~mkl +fftw'
   gsibec:
     require::
     - '@1.2.1 ~mkl'

--- a/configs/sites/tier1/s4/packages.yaml
+++ b/configs/sites/tier1/s4/packages.yaml
@@ -31,7 +31,7 @@ packages:
   # Remove this section to switch to intel-oneapi-mkl
   ectrans:
     require::
-    - '@1.5.0 ~mkl +fftw'
+    - '@1.2.0 ~mkl +fftw'
   gsibec:
     require::
     - '@1.2.1 ~mkl'

